testing/py-google-apputils/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=py-google-apputils
+_pkgname=google-apputils
+pkgver=0.4.0
+pkgrel=0
+pkgdesc="Google Application Utilities for Python"
+url="https://github.com/google/google-apputils"
+arch="noarch"
+license="Copyright 2014, Google Inc. All rights reserved."
```

Comment:
This is semantically and even factually incorrect. google-apputils is licensed under [Apache License 2.0](https://github.com/google/google-apputils/blob/master/LICENSE), so there should be `license="ASL-2.0"`.